test/schema/fixtures/config.schema.test.failure/root.aliases_missing_secrets.yaml

```
@@ -0,0 +1,10 @@
+selfservice:
+  default_browser_return_url: "#/definitions/defaultReturnTo"
+
+dsn: foo
+
```

This should fail if secrets.cipher is unset:

```suggestion

secrets:
  default: some-secret

```